[network_events.py]
  expected: TIMEOUT
  [test_same_navigation_id]
    expected: FAIL

  [test_same_request_id]
    expected: FAIL

  [test_subscribe_to_one_context]
    expected: FAIL
